data/Kaif987.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Mohammad Kaif",
+  "type": "personal",
+  "bio": "A MERN stack developer who loves programming",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/mohammad-kaif-4a0ab2231/",
+      "icon": "FaLinkedinr"
```

Comment:
The current icon seems to be invalid. You might want to consider replacing it with **'FaLinkedin'** instead.
You can also refer to [ICONS](https://linkfree.eddiehub.io/icons) , thanks :+1: 
```Suggestion
      "icon": "FaLinkedin"
```